app/main.py

```diff
@@ -1,14 +1,22 @@
 def get_human_age(cat_age: int, dog_age: int) -> list:
-    cat_to_human = convert_to_human(cat_age, 15, 9, 4)
-    dog_to_human = convert_to_human(dog_age, 15, 9, 5)
-    return [cat_to_human, dog_to_human]
+    if cat_age < 0 or dog_age < 0:
+        raise ValueError("Age cannot be negative")
+    return [
+        convert_to_human(cat_age, 15, 9, 4),
+        convert_to_human(dog_age, 15, 9, 5)
+    ]
 
 
 def convert_to_human(
-        animal_age: int, first_year: int, second_year: int, each_year: int
+        animal_age: int,
+        first_year: int,
+        second_year: int,
+        each_year: int
 ) -> int:
+    if animal_age < 0:
+        return 0
     if animal_age < first_year:
         return 0
     if animal_age < first_year + second_year:
         return 1
-    return 2 + (animal_age - first_year - second_year) // each_year
+    return 2 + (animal_age - (first_year + second_year)) // each_year
```

Comment:
The calculation of `remaining_years` in the return statement should be enclosed in parentheses to ensure the correct order of operations. The subtraction should be fully enclosed to avoid precedence issues: `2 + ((animal_age - (first_year + second_year)) // each_year)`.